packages/project-utils/bundling/app/config/webpack.config.js

```diff
@@ -234,13 +234,16 @@ module.exports = function(webpackEnv, { paths, babelCustomizer }) {
                             safari10: true
                         },
                         // Added for profiling in devtools
+                        /* eslint-disable-next-line @typescript-eslint/camelcase */
```

Comment:
Is it possible to apply the `/* eslint-disable-next-line @typescript-eslint/camelcase */` rule just once, at the top of the file? Just curious.